clang/test/SemaSYCL/sycl-dllimport-dllexport.cpp

```diff
@@ -0,0 +1,50 @@
+// RUN: %clang_cc1 -triple x86_64-pc-linux-gnu -fsycl-is-device -fms-extensions -DTEST %s -o /dev/null 2>&1 | FileCheck %s --check-prefixes CHECK
+
+// RUN: %clang_cc1 -triple spir64-unknown-windows-sycldevice -fms-extensions -DTEST %s -o /dev/null 2>&1 | FileCheck %s --check-prefixes CHECK
+
+// RUN: %clang_cc1 -triple spir64-unknown-windows-sycldevice -fsycl-is-device -fms-extensions -DTEST %s -o /dev/null 2>&1 | FileCheck %s --check-prefixes CHECKALL
+
+// RUN: %clang_cc1 -triple spir64-unknown-windows-sycldevice -fsycl-is-device -fms-extensions -fsyntax-only -verify  -DTEST2 %s
+
+#if TEST
+// CHECK: warning: __declspec attribute 'dllexport' is not supported
+int __declspec(dllexport) foo(int a)
+{
+  return a;
+}
+
+// CHECK: warning: __declspec attribute 'dllimport' is not supported
+int __declspec(dllimport) bar();
+
+// CHECK: warning: unknown attribute 'dllimport' ignored
+int [[dllimport]]xoo();
+
+// CHECKALL: warning: unknown attribute 'dllimport' ignored
+int zoo() __attribute__((dllimport));
+
+#elif TEST2
+
+// emit error if dllimport function is called in sycl kerenal
```

Comment:
// emit error if dllimport function is called in sycl kernel

Reply:
Opps.  Sorry.  Changed.